backend/docker.go

```diff
@@ -29,6 +29,7 @@ var (
 		"CERT_PATH":       "directory where ca.pem, cert.pem, and key.pem are located (default \"\")",
 		"CMD":             "command (CMD) to run when creating containers (default \"/sbin/init\")",
 		"MEMORY":          "memory to allocate to each container (0 disables allocation, default \"4G\")",
+		"SHM":             "/dev/shm to allocate to each container (0 disables allocation, default \"64MB\")",
```

Comment:
There seems to be confusion/inconsistency about this in the docker source and docs, too, where it is claimed that the default is `"64MB"`, but the numeric value is `67108864` (`1024 * 1024 * 64`).  Docker appears to support (via https://github.com/docker/go-units) `MB`, `M`, and `m`, but *not* `MiB`:

``` bash
$ docker version --format='{{.Client.Version}}'
1.12.0
$ docker run --rm -it --shm-size=64MB alpine:latest df -h /dev/shm
Filesystem                Size      Used Available Use% Mounted on
shm                      64.0M         0     64.0M   0% /dev/shm
$ docker run --rm -it --shm-size=64M alpine:latest df -h /dev/shm
Filesystem                Size      Used Available Use% Mounted on
shm                      64.0M         0     64.0M   0% /dev/shm
$ docker run --rm -it --shm-size=64m alpine:latest df -h /dev/shm
Filesystem                Size      Used Available Use% Mounted on
shm                      64.0M         0     64.0M   0% /dev/shm
$ docker run --rm -it --shm-size=64MiB alpine:latest df -h /dev/shm
docker: invalid size: '64MiB'.
See 'docker run --help'.
```

How about we switch to the slightly more ambiguous yet more consistent with Docker `m`/`M`?